galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/LocalEcosystemImpl.java

```diff
@@ -83,6 +83,7 @@
     private String galasaBootVersion;
     private String simplatformVersion;
     private Path mavenLocal;
+    Path isolatedRepoDirectory;
```

Comment:
why not add "private" so it lines up with the rest

Reply:
Quite right. Changes made. Trying to be too speedy.